### PR TITLE
Bug/8429 award obligations hover

### PR DESCRIPTION
**Pull request by brianpetway**
**High level description:**

For the donut chart on the Agency page, move the labels to below the chart and make the tooltips show when hovering on them

**Technical details:**

I used the equations that were already there to move the labels, and I made them appear all on one line. I copied the d3 methods used for the hover on the chart and put them on the labels.

**JIRA Ticket:**
[DEV-8429](https://federal-spending-transparency.atlassian.net/browse/DEV-8429)

**Mockup:**
https://app.zeplin.io/project/61f806d9ad825610f7b1fc0a/screen/6206d4a0f4c33511d4600adc

The following are ALL required for the PR to be merged:

Author:
- [x ] Linked to this PR in JIRA ticket
- [ ] Scheduled demo including Design/Testing/Front-end OR Provided instructions for testing in JIRA and PR `if applicable`
- [x ] Verified cross-browser compatibility: Chrome, Safari, Firefox, Edge
- [x ] Verified mobile/tablet/desktop/monitor responsiveness
- [ ] Verified that this PR does not create any *new* accessibility issues (via Axe Chrome extension)
- [ ] Added Unit Tests for helper functions, reducers, models and Container/Component Interactivity Expectations `if applicable` [React Testing Library](react-testing-library.md)
- [ ] [API contract](https://github.com/fedspendingtransparency/usaspending-api/tree/dev/usaspending_api/api_contracts) updated `if applicable`
- [ ] [Component Library Integration Status Report](https://github.com/fedspendingtransparency/data-act-documentation/blob/data-transparency-ui/frontend_apps/component-library-integration-status.md) updated `if applicable`

Reviewer(s):
- [ ] Design review complete `if applicable`
- [ ] [API #1234](https://github.com/fedspendingtransparency/usaspending-api/pull/1234) merged concurrently `if applicable`
- [ ] Code review complete

### CODE REVIEW

File: src/_scss/pages/agency/visualizations/_obligationsByAwardType.scss [hunk 1]
```diff
@@ -42,9 +40,6 @@
             font-weight: $font-semibold;
         }
         .obligations-by-award-type__donut {
-
-
-
```
Comment (nick-torres) [inline src/_scss/pages/agency/visualizations/_obligationsByAwardType.scss:47]: my name is brian and i hate whitespace

Comment (brianpetway) [reply src/_scss/pages/agency/visualizations/_obligationsByAwardType.scss:47]: TRUE

File: src/js/containers/agency/visualizations/ObligationsByAwardTypeContainer.jsx [hunk 2]
```diff
@@ -35,7 +35,7 @@ export default function ObligationsByAwardTypeContainer({ fiscalYear, windowWidt
             obligationsByAwardTypeRequest.current.cancel();
         }
         dispatch(resetAwardObligations());
-    }, []);
+    }, [dispatch]);
```
Comment (nick-torres) [inline src/js/containers/agency/visualizations/ObligationsByAwardTypeContainer.jsx:38]: do you want this one there too?

Comment (brianpetway) [reply src/js/containers/agency/visualizations/ObligationsByAwardTypeContainer.jsx:38]: That one doesn't seem to be affecting anything. The others were breaking things, so I added the ignore line.

**Pull request merged**